ejb/WFLY-6892-ejb-access-logging.adoc

```diff
@@ -0,0 +1,309 @@
+                    <xs:annotation>
+                        <xs:documentation>
+                            The value to be used to indicate the end of a record. If set to null no delimiter will be used
+                            at the end of the record. The default value is a line feed.
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+                <xs:element name="meta-data" type="propertiesType" minOccurs="0">
+                    <xs:annotation>
+                        <xs:documentation>
+                            Sets the meta data to use in the structured format. Properties will be added to each log message.
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+            </xs:all>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="pattern" type="xs:string" use="required"/>
+            <xs:attribute name="date-format" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation>
+                        The date/time format pattern. The pattern must be a valid
+                        java.time.format.DateTimeFormatter.ofPattern() pattern.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="pretty-print" type="xs:boolean" default="false">
+                <xs:annotation>
+                    <xs:documentation>
+                        Indicates whether or not pretty printing should be used when formatting.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="zone-id" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation>
+                        The zone ID for formatting the date and time. The system default is used if left undefined.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:attribute>
+        </xs:complexType>
+
+----
+
+=== Nice-to-Have Requirements
+
+* users should be able to configure condition/filter to determine which requests to log
+
+=== Non-Requirements
+
+* configure ejb access logging from management console
+
+
+//== Implementation Plan
+////
+Delete if not needed. The intent is if you have a complex feature which can 
+not be delivered all in one go to suggest the strategy. If your feature falls 
+into this category, please mention the Release Coordinators on the pull 
+request so they are aware.
+////
+== Test Plan
+
+== Community Documentation
+////
+Generally a feature should have documentation as part of the PR to wildfly master, or as a follow up PR if the feature is in wildfly-core. In some cases though the documentation belongs more in a component, or does not need any documentation. Indicate which of these will happen.
```

Comment:
@chengfang this part is not yes complete

Reply:
Will work on it once the feature is implemented and finalized.